show-memory-limit.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+const v8 = require('v8');
+
+console.log(v8.getHeapStatistics());
```

Comment:
never seen this before, but looks useful

Reply:
Yeah I was just curious how much memory I was using.